mrjob/aws.py

```diff
@@ -208,7 +225,13 @@ def _boto3_paginate(what, boto3_client, api_call, **api_params):
     _delay = api_params.pop('_delay', None)
 
     paginator = boto3_client.get_paginator(api_call)
```

Comment:
Is the passed in boto3_client passed in already wrapped?  I feel like it was when you showed me this before would be handy to have a note about the fact that you have doubled the wrapping.

Reply:
Yes it is wrapped. The `paginator = boto3_client.get_paginator(api_call)` will perform the retry, which is useless.